sklearn/utils/enet_proj.py

```diff
@@ -0,0 +1,59 @@
+__author__ = 'arthur'
```

Comment:
Please follow the conventions in other headers of files of the project to state authorship consistently.